command/output.go

```diff
@@ -68,6 +73,20 @@ func (c *OutputCommand) Run(args []string) int {
 		return 1
 	}
 
+        if allOutputs {
```

Comment:
It may be shorter code to not have the allOutputs var, but I think it is clearer what is happening with a separate one.  I'll defer to your judgement.